marlowe-website/.gitignore

```diff
@@ -0,0 +1,3 @@
+public/js
+public/css
+public/index.html
```

Comment:
why not just `public/`?

Reply:
I think when we upload images those will be in the `public` folder and we'd like to commit that. But will see